vic_gtests/TestKinematics.cpp

```diff
@@ -68,6 +71,65 @@ TEST(TestKinematics, EulerAngles)
     // which is not Sequential
 }
 
+TEST(TestKinematics, Quaternion)
+{
+    const double eps = 1e-10;
```

Comment:
does this test make sure that each of the if..else blocks is tested? i think right now we are only sure that one of the blocks works

Reply:
new version does.